README.md

```diff
@@ -1,35 +1,92 @@
-# Fether
+# Parity Fether - a fast and decentralized wallet based on a light-client
 
 ## Get the latest binary
 
-https://github.com/paritytech/fether/releases
+[» Download the latest release «](https://github.com/paritytech/fether/releases)
 
-## Getting started
+
+### Join the chat!
+
+Get in touch with us on Gitter:
+[![Gitter: Parity](https://img.shields.io/badge/gitter-parity-4AB495.svg)](https://gitter.im/paritytech/parity)
```

Comment:
Shall we use a different Gitter for UI stuff?

Reply:
yes, I think this would be good, and help separate things a little.